### ISSUE TITLE
Login Fails

### ISSUE THREAD

**Issue by WisdomCode** [eddie3/gogrepo#38]
For a few days now, there seems to be an issue in several Ways:
First, the updating throwed a 403 at me, tried with python2 and python3 on a debian 9 system. 
Because I assumed that this may be due to the cookies being expired, I deleted the cookies.dat file. Now, when I try to login with the correct credentials, it fails. Asking me whether Username and password are correct (which they are, after a while I wrote them into the variables directly inside the script to be sure of that to no effect). I'm rather skilled with python, however not with the workings of gog.com, so I may assist you debugging if needed.

**Comment by drcursor** [comment]
I'm having the same issue, but was able to get gogrepo working by following https://github.com/eddie3/gogrepo/issues/17#issuecomment-248807909

**Comment by WisdomCode** [comment]
Good Idea, I tried that before however (well, I used firefox, with the cookies.txt plugin, they are saved in the netscape format, which should work) with no success. Script fails, saying "failed to load cookies, did you login first?".

**Comment by WisdomCode** [comment]
I was able to restore the cookie from a different system, using a freshly downloaded gogrepo.py file. Strange enough, I also needed to redownload the script on the old system to use this cookie. For the record I should add that the script has been the same for 4 months and that I my previous script was identical to it. If anyone has an idea how that came to be I would be interested, however I think this issue should be closed then, the error 403 should be further discussed under Issue #39
